server/controllers/courseCtrl.js

```diff
@@ -482,6 +482,16 @@ async function getIndex(req, res, next) {
 
   const courseCode = req.params.courseCode.toUpperCase()
   const lang = languageUtils.getLanguage(res) || 'sv'
+  let klaroConsentCookie = false
+  if (req.cookies.klaro) {
+    const consentCookiesArray = req.cookies.klaro.slice(1, -1).split(',')
+    // eslint-disable-next-line prefer-destructuring
+    const analyticsConsentCookieString = consentCookiesArray
+      .find(cookie => cookie.includes('analytics-consent'))
+      .split(':')[1]
+    // eslint-disable-next-line no-const-assign
+    klaroConsentCookie = analyticsConsentCookieString === 'true'
```

Comment:
Because it's specifically analytics consent, it is better to rename `klaroConsentCookie` to `klaroAnalyticsConsentCookie`. (In case in future we will need other type of consent cookie.)